backend/schema/metadatatypemap.go

```diff
@@ -0,0 +1,34 @@
+package schema
+
+import (
+	"fmt"
+
+	schemapb "github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1/schema"
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+type MetadataTypeMap struct {
+	Pos Position `parser:"" protobuf:"1,optional"`
+
+	Runtime    string `parser:"'+' 'typemap' @('go' | 'kotlin')" protobuf:"2"`
```

Comment:
We might do this in other places, but I wonder if it will be tough to know where we need to add new languages to FTL. Like maybe we need to have `rust` in here too ? :)